<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MatterWindow</class>
 <widget class="QMainWindow" name="MatterWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>937</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Matter Details</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QMainWindow#MainWindow{
	
	font: 10pt &quot;Microsoft Tai Le&quot;;
	background-color:rgb(226, 226, 226);
}
QGroupBox{
	font: 75 12pt &quot;Microsoft Tai Le&quot;;
}
QPushButton{
	
	font: 75 10pt &quot;Microsoft Tai Le&quot;;
	color: rgb(202, 161, 56);
	background-color: rgb(11, 56, 97);
}
QToolButton{
	font: 75 10pt &quot;Microsoft Tai Le&quot;;
}
QToolButton#prevPage{
	
	font: 75 10pt &quot;Microsoft Tai Le&quot;;
	color: rgb(202, 161, 56);
	background-color: rgb(11, 56, 97);
}
QToolButton#nextPage{
	
	font: 75 10pt &quot;Microsoft Tai Le&quot;;
	color: rgb(202, 161, 56);
	background-color: rgb(11, 56, 97);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="styleSheet">
         <string notr="true">font: 75 12pt &quot;Microsoft Tai Le&quot;;</string>
        </property>
        <property name="text">
         <string>Matter #</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="clientNum">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_11">
        <property name="styleSheet">
         <string notr="true">font: 75 12pt &quot;Microsoft Tai Le&quot;;</string>
        </property>
        <property name="text">
         <string>.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="matterNum">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGroupBox" name="matterGroup">
        <property name="title">
         <string>Matter Info</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>1</number>
         </property>
         <property name="leftMargin">
          <number>1</number>
         </property>
         <property name="topMargin">
          <number>1</number>
         </property>
         <property name="rightMargin">
          <number>1</number>
         </property>
         <property name="bottomMargin">
          <number>1</number>
         </property>
         <item>
          <widget class="QFrame" name="frame">
           <property name="styleSheet">
            <string notr="true">QFrame{ background-color: rgb(248, 233, 210); }</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::Panel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QFormLayout" name="formLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Attorney Initials</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="attorneyInitials"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Matter Type</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="matterType"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Date Opened</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QDateEdit" name="dateOpened">
              <property name="showGroupSeparator" stdset="0">
               <bool>false</bool>
              </property>
              <property name="calendarPopup">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Date Closed</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QDateEdit" name="dateClosed">
              <property name="showGroupSeparator" stdset="0">
               <bool>false</bool>
              </property>
              <property name="calendarPopup">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Box Number</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLineEdit" name="boxNumber"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="billingGroup">
        <property name="title">
         <string>Billing Info</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="spacing">
          <number>1</number>
         </property>
         <property name="leftMargin">
          <number>1</number>
         </property>
         <property name="topMargin">
          <number>1</number>
         </property>
         <property name="rightMargin">
          <number>1</number>
         </property>
         <property name="bottomMargin">
          <number>1</number>
         </property>
         <item>
          <widget class="QToolButton" name="useClientAddress">
           <property name="text">
            <string>Same as Client</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_2">
           <property name="styleSheet">
            <string notr="true">QFrame{ background-color: rgb(248, 233, 210); }</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::Panel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QGridLayout" name="gridLayout">
              <item row="2" column="0">
               <widget class="QLineEdit" name="firstName"/>
              </item>
              <item row="2" column="2">
               <widget class="QLineEdit" name="lastName"/>
              </item>
              <item row="2" column="1">
               <widget class="QLineEdit" name="middleInitial">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>75</width>
                  <height>16777215</height>
                 </size>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>LastName</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>First Name</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>Middle Initial</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QFormLayout" name="formLayout">
              <item row="0" column="0">
               <widget class="QLabel" name="label_27">
                <property name="text">
                 <string>Address 1</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="addr1"/>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_28">
                <property name="text">
                 <string>Address 2</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="addr2"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QGridLayout" name="gridLayout_10">
              <item row="0" column="0">
               <widget class="QLabel" name="label_29">
                <property name="text">
                 <string>City</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLabel" name="label_30">
                <property name="text">
                 <string>State</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QLabel" name="label_31">
                <property name="text">
                 <string>Zip</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLineEdit" name="billCity"/>
              </item>
              <item row="1" column="1">
               <widget class="QComboBox" name="billState"/>
              </item>
              <item row="1" column="2">
               <widget class="QLineEdit" name="billZip">
                <property name="maximumSize">
                 <size>
                  <width>75</width>
                  <height>16777215</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QGroupBox" name="documentGroup">
        <property name="title">
         <string>Original Documents</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QToolButton" name="attachDocument">
             <property name="text">
              <string>Add New Document</string>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>Icons/plus.ico</normaloff>Icons/plus.ico</iconset>
             </property>
             <property name="toolButtonStyle">
              <enum>Qt::ToolButtonTextBesideIcon</enum>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableWidget" name="documentList">
           <column>
            <property name="text">
             <string>Name</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>FileDirectory</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>View</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="adversePartyGroup">
        <property name="title">
         <string>Adverse Parties</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QFrame" name="frame_3">
           <property name="styleSheet">
            <string notr="true">QFrame{ background-color: rgb(248, 233, 210); }</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::Panel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="1" column="0">
               <widget class="QLabel" name="label_14">
                <property name="text">
                 <string>First Name</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLineEdit" name="apFirst"/>
              </item>
              <item row="1" column="2">
               <widget class="QLabel" name="label_16">
                <property name="text">
                 <string>LastName</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLabel" name="label_15">
                <property name="text">
                 <string>Middle Initial</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLineEdit" name="apMiddle">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>75</width>
                  <height>16777215</height>
                 </size>
                </property>
               </widget>
              </item>
              <item row="2" column="2">
               <widget class="QLineEdit" name="apLast"/>
              </item>
              <item row="0" column="0" colspan="3">
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QToolButton" name="apNew">
                  <property name="text">
                   <string>New</string>
                  </property>
                  <property name="icon">
                   <iconset>
                    <normaloff>Icons/plus.ico</normaloff>Icons/plus.ico</iconset>
                  </property>
                  <property name="toolButtonStyle">
                   <enum>Qt::ToolButtonTextBesideIcon</enum>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QToolButton" name="apSave">
                  <property name="text">
                   <string>Save</string>
                  </property>
                  <property name="icon">
                   <iconset>
                    <normaloff>Icons/save.ico</normaloff>Icons/save.ico</iconset>
                  </property>
                  <property name="toolButtonStyle">
                   <enum>Qt::ToolButtonTextBesideIcon</enum>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QToolButton" name="apClear">
                  <property name="text">
                   <string>Clear</string>
                  </property>
                  <property name="icon">
                   <iconset>
                    <normaloff>Icons/clear.ico</normaloff>Icons/clear.ico</iconset>
                  </property>
                  <property name="toolButtonStyle">
                   <enum>Qt::ToolButtonTextBesideIcon</enum>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QToolButton" name="apDelete">
                  <property name="text">
                   <string>Delete</string>
                  </property>
                  <property name="icon">
                   <iconset>
                    <normaloff>Icons/delete.ico</normaloff>Icons/delete.ico</iconset>
                  </property>
                  <property name="toolButtonStyle">
                   <enum>Qt::ToolButtonTextBesideIcon</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label_17">
                <property name="text">
                 <string>Reason</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="reason"/>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="partyList">
           <column>
            <property name="text">
             <string>First Name</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>M.I.</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Last Name</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Reason</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSave"/>
   <addaction name="actionEdit"/>
   <addaction name="actionClose"/>
  </widget>
  <action name="actionSave">
   <property name="icon">
    <iconset>
     <normaloff>Icons/save.ico</normaloff>Icons/save.ico</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="icon">
    <iconset>
     <normaloff>Icons/note.ico</normaloff>Icons/note.ico</iconset>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset>
     <normaloff>Icons/arrow-1.ico</normaloff>Icons/arrow-1.ico</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
